### PR TITLE
Update release process description

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: Builds.md [hunk 1]
```diff
@@ -119,251 +119,82 @@ When major or minor releases are done, there should be a time period of at least
...
-1. Push changes to origin
-   * `./gitAll push --tags origin $BRANCHNAME`
-1. Once all previous builds are successful, trigger the ['xtext-release' build job](https://ci.eclipse.org/xtext/job/xtext-release/) with the release version and branch name as parameters.
-1. Delete the release branches
-   * `./gitAll branch -D $BRANCHNAME`
-   * `./gitAll push --delete origin $BRANCHNAME`
-1. Promote staged release on [oss.sonatype.org](https://oss.sonatype.org). Can only be done by Xtext release engineer (@kthoms, @spoenemann, @dhuebner)
+To initiate a release build start the build job [release-prepare-branches](https://ci.eclipse.org/xtext/job/releng/job/release-prepare-branches/). This job automates the process to a large degree. Once it is triggered it takes about 4-5 hours to complete. If you are interested in the details, read the section "What happens when the _release-prepare-branches_ job is executed".
+
+After everything above has run smooth, it is finally time for some manual steps again. We will try to integrate some of them into the automated release pipeline from time to time.
```
Comment (Hejado) [inline Builds.md]: smooth => smoothly

File: Builds.md [hunk 2]
```diff
@@ -119,251 +119,82 @@ When major or minor releases are done, there should be a time period of at least
...
+
+1. Check that everything was promoted correctly:
+   * [Xtext Downloads Page](https://www.eclipse.org/modeling/tmf/downloads/) should list the new release
+   * Milestone / Release Candidate Builds will be listed in _Stable Builds_, Release Builds below _Latest Releases_ (might need to expand other releases)
+2. Delete the release branches. Go to `xtext-umbrella` and run
```
Comment (Hejado) [inline Builds.md]: If we merge eclipse/xtext-umbrella#184, we can remove this step

File: Builds.md [hunk 3]
```diff
@@ -119,251 +119,82 @@ When major or minor releases are done, there should be a time period of at least
...
+   ```./gitAll push --delete origin $BRANCHNAME```
+3. Promote staged release on [oss.sonatype.org](https://oss.sonatype.org). Can only be done by Xtext release engineer (@kthoms, @spoenemann, @Hejado)
```
Comment (Hejado) [inline Builds.md]: GitHub automatically numbers lists if all points start with `1.`.

File: Builds.md [hunk 4]
```diff
@@ -119,251 +119,82 @@ When major or minor releases are done, there should be a time period of at least
...
    * Twitter
    * Blog (for releases)
-1. Add / Update xtext-reference-projects
-   * Only applicable for releases or the first milestone of a new release phase.
-1. Adjust the bootstrap version to use the newly produced milestone / release
+8. Add / Update xtext-reference-projects (Only on final release)
+9. Adjust the bootstrap version to use the newly produced milestone / release
+10. xtext-apidiff (Only on final release)
+    * adapt https://github.com/xtext/xtext-apidiff/blob/master/create-api-diff.sh
+    * adapt Jenkins Configuration https://github.com/xtext/xtext-apidiff/blob/master/CBI.Jenkinsfile
```
Comment (Hejado) [inline Builds.md]: CBI.Jenkinsfile => 404

File: Builds.md [hunk 5]
```diff
@@ -119,251 +119,82 @@ When major or minor releases are done, there should be a time period of at least
...
    * Newsgroup / Forum
    * Mailing list
    * Gitter
    * Twitter
    * Blog (for releases)
-1. Add / Update xtext-reference-projects
-   * Only applicable for releases or the first milestone of a new release phase.
-1. Adjust the bootstrap version to use the newly produced milestone / release
+8. Add / Update xtext-reference-projects (Only on final release)
+9. Adjust the bootstrap version to use the newly produced milestone / release
```
Comment (Hejado) [inline Builds.md]: Add reference to bot-updates job

**Comment by kthoms** [comment]
@Hejado Thanks for the review. Updates the PR accordingly.

**Pull request merged**